src/components/QuicksightDashboardPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Store } from "redux";
+import { RootState } from "../store";
+import * as PropTypes from "prop-types";
+import Header from "./Header";
+import Footer from "./Footer";
+import title from "../utils/title";
+import QuicksightDashboard from "./QuicksightDashboard";
+
+export interface QuicksightDashboardPageProps
+  extends React.Props<QuicksightDashboardPageProps> {
+  params: {
+    library?: string;
+  };
+}
+
+export interface QuicksightDashboardPageContext {
+  editorStore: Store<RootState>;
+}
+
+/** Page holds quicksight dashboards. */
+export default class QuicksightDashboardPage extends React.Component<
+  QuicksightDashboardPageProps
+> {
+  context: QuicksightDashboardPageContext;
+
+  static contextTypes: React.ValidationMap<QuicksightDashboardPageContext> = {
+    editorStore: PropTypes.object.isRequired as React.Validator<Store>,
+  };
+
+  static childContextTypes: React.ValidationMap<object> = {
+    library: PropTypes.func,
+  };
+
+  getChildContext() {
+    return {
+      library: () => this.props.params.library,
+    };
+  }
+
+  render(): JSX.Element {
+    const { library } = this.props.params;
+    return (
+      <div className="quicksight-dashboard">
+        <Header />
+        <main className="body">
+          <QuicksightDashboard dashboardId="library" />
```

Comment:
Again, we don't need to pass this hardcoded `dashboarId` prop here. We can just use the hardcoded value directly in the child component.

Reply:
I can remove this - but it was my assumption that the admin interface may provide more than one dashboard depending on the access level of the user.